tests/test_wide_book_trial.py

```
@@ -0,0 +1,343 @@
+    def __init__(self, payload):
+        self._payload = payload
+
+    def json(self):
+        return self._payload
+
+
+class _FakeSession:
+    """Serves the two books `evaluate` fetches, 13c wide on both legs."""
+
+    def get(self, url, params=None, timeout=None):
+        up = params["token_id"] == "tok_up"
```

Ruff (0.16.3)

[error] Possible hardcoded password assigned to: "token_id"

(S105)